src/pudl/transform/eia861.py

```diff
@@ -1405,10 +1416,13 @@ def demand_side_management(tfr_dfs):
         transformed_dsm2.drop(
             ee_cols
             + dr_cols
-            + program_cols
```

Comment:
how come the `program_cols` are kept in with the misc df? if they are in with `dsm_ee_dr` above?

Reply:
oops, I think that's a mistake